projects/smartseq2/CreateSs2AdapterMetadata.wdl

```diff
@@ -105,14 +105,8 @@ workflow CreateSs2AdapterMetadata {
   String multi_sample_pipeline_version = ParseSs2CromwellMetadata.pipeline_version
   String single_sample_pipeline_version = ParseSs2CromwellMetadata.single_sample_pipeline_version
 
-  call Tasks.GetSs2PipelineVersion as CheckPipelineVersion {
```

Comment:
I have a vague memory of @nikellepetrillo mentioning we run some tests just as fail safe tasks that will fail if something is off. But I also might be thinking of project_name... Before we delete this I just want to make sure it's not necessary anymore?

Reply:
yup it was project_name, i think this can be removed!

Reply:
yeah i think we added this just copying off of Optimus....Optimus needs this task because we actually need to append the `1.0.0` pipeline_version from MergeLooms to the `v_`

Since we are grabbing both the versions from the metadata.json in ss2 then we can get rid of it